account_invoice_rounding_by_currency/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================================
+Unit rounded invoice (Swedish rounding) by currency
+===================================================
+
+This module extends functionality of module `Unit rounded invoice <https://github.com/OCA/account-invoicing/tree/8.0/account_invoice_rounding>`_.
+
+It allows to set, in accounting settings, a rounding precision for each currency,
+such as 0.05 CHF for Swiss invoices
+
+Configuration
+=============
+
+In Settings -> Configuration -> Accounting you will find
+Currencies Rounding Rules
+Set currency rule for each currency you need to handle.
+
+- `Swedish Round globally`
+
+  To round your invoice total amount, this option will do the adjustment in
+  the most important tax line of your invoice.
+
+- `Swedish Round by adding an invoice line`
+
+  To round your invoice total amount, this option create a invoice line without
```

Comment:
To round your invoice total amount, this option create a invoice line this option will create an invoice line without tax on it.
This invoice line is tagged as `is_rounding`.
(Period=".")